setup.py

```diff
@@ -133,8 +133,6 @@ def get_flash_attention_extensions(cuda_version: int, extra_compile_args):
                 "nvcc": extra_compile_args.get("nvcc", [])
                 + [
                     "-O3",
-                    "-U__CUDA_NO_HALF_OPERATORS__",
-                    "-U__CUDA_NO_HALF_CONVERSIONS__",
```

Comment:
So this wasn't needed in the end?

Reply:
Nope, I just copy pasted it because it was in cutlass's build process I think. Good catch